.github/workflows/push.yaml

```diff
@@ -2,603 +2,21 @@ name: push
-        run: |
-          if [ ! -d "./tmp/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}/operators/${{ env.REPOSITORY_NAME }}" ]; then
-            export PR_REQUEST_BODY="$(cat config/community-operators/pr-first-release-body.txt)"
-          else
-            export PR_REQUEST_BODY="$(cat config/community-operators/pr-body.txt)"
-          fi
-          PR_REQUEST_BODY="${PR_REQUEST_BODY//'%'/'%25'}"
-          PR_REQUEST_BODY="${PR_REQUEST_BODY//$'\n'/'%0A'}"
-          PR_REQUEST_BODY="${PR_REQUEST_BODY//$'\r'/'%0D'}"
-          echo "::set-output name=pull-request-body::"$(echo $PR_REQUEST_BODY)""
-
-      - name: Download Dist Directory
-        uses: actions/download-artifact@v2
-        with:
-          name: dist
-          path: dist
-
-      - name: Untar Previously Created Bundle
-        shell: bash
-        run: |
-          # Create Bundle Directory
-          mkdir bundle
-
-          # Untar
-          ls dist/${{ env.REPOSITORY_NAME }}-bundle*.tar.gz | xargs -n1 tar -C bundle/ --strip-components=1 -xzvf
-
-      - name: Create and Copy Bundle to Community Operators
-        shell: bash
-        run: |
-          sed -i '/replaces: '"${{ env.REPOSITORY_NAME }}"'/d' ./bundle/manifests/${{ env.REPOSITORY_NAME }}.clusterserviceversion.yaml
-          mkdir -p ./tmp/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}/operators/${{ env.REPOSITORY_NAME }}/${{ env.BUNDLE_VERSION }}
-          /bin/cp -v -R ./bundle/* ./tmp/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}/operators/${{ env.REPOSITORY_NAME }}/${{ env.BUNDLE_VERSION }}
-          /bin/cp -v -R ./config/community-operators/ci.yaml ./tmp/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}/operators/${{ env.REPOSITORY_NAME }}
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
-        env:
-          PR_ACTOR: raffaele.spazzoli@gmail.com
-        with:
-          path: ./tmp/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}
-          commit-message: ${{ env.REPOSITORY_NAME }} release ${{ env.BUNDLE_VERSION }}
-          committer: ${{ github.actor }} <${{ env.PR_ACTOR }}>
-          author: ${{ github.actor }} <${{ env.PR_ACTOR }}>
-          signoff: true
-          branch: ${{ env.REPOSITORY_NAME }}-${{ env.BUNDLE_VERSION }}
-          delete-branch: true
-          push-to-fork: ${{ github.repository_owner }}/${{ env.COMMUNITY_OPERATORS_REPOSITORY_NAME }}
-          title: ${{ env.REPOSITORY_NAME }} new version ${{ env.BUNDLE_VERSION }}
-          body: |
-            ${{ steps.set-community-operators-parameters.outputs.pull-request-body }}
-          token: ${{ secrets.COMMUNITY_OPERATOR_PAT }}
+  shared-operator-workflow:
+    name: shared-operator-workflow
+    uses: redhat-cop/github-workflows-operators/.github/workflows/release-operator.yml@v1.0.0
+    secrets:
+      COMMUNITY_OPERATOR_PAT: ${{ secrets.COMMUNITY_OPERATOR_PAT }}
+      BUNDLE_IMAGE_REPOSITORY: ${{ secrets.BUNDLE_IMAGE_REPOSITORY }}
```

Comment:
mm I don't like this. the repo were autodetected from the repo name. And at any rate they should be variables not secrets.

Reply:
Will incorporate into shared workflow and update here shortly